BOUNTY_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+The following bug bounty submission template. For details see the blog post at https://medium.com/fluidity/... For a prompt response, create an issue and include the following content in an email to our team at bounty@airswap.io.
```

Comment:
For a prompt response, include the following content in an email to our team at bounty@airswap.io.